snippets/getMeridiemOfInteger.md

```diff
@@ -0,0 +1,15 @@
+### toMeridiem
+
+Uses modulo operator (`%`) to transform an integer to a 12 hour clock format. Stringifies transformed integer with concatenated meridiem suffix. Maintains 12 hour format principles with conditional check (0am - 12am).
+
+```js
+const toMeridiem = num => {
+    const meridiems = ["am", "pm"];
+    let period = num > 11 ? 1 : 2;
```

Comment:
Using an array and a reference is error prone if there are few cases. Being explicit here is better:
```js
let period = num > 11 ? 'pm' : 'am'
```

As a case in point: JS uses 0 index, you used 1 index leading to either `XXam` or `XXundefined`

Reply:
You're right, that slipped my attention. I rewrote how it quickly to keep it simpler. I'll fix that.